skycultures/arabic_lunar_stations/dso_names.fab

```diff
@@ -0,0 +1,13 @@
+#NGC 869 |_("المعصم 1")
+#NGC 884 |_("المعصم 2") 
+#M 45    |_("الثريا") 
+#M 44    |_("النثرة")
+#M 7     |_("إبرة العقرب")
+NGC 869 |_("The Wrist 1")
+NGC 884 |_("The Wrist 2")
+NGC 869 |_("The She-Camel's Brand 1")
+NGC 884 |_("The She-Camel's Brand brand 2")
```

Comment:
Is it correct?

Reply:
Yes.
I started including references in the constellation names, but I got  [issue 2055](https://github.com/Stellarium/stellarium/issues/2055).
I will add the references since this issue is resolved and test with  0.21.2-07bea75.

Reply:
I think the question was about "Brand brand" (repeated word)

Reply:
Thanks @gzotti 
Fixed.